tests/plugins/copyformatting/undointegration.js

```diff
@@ -0,0 +1,83 @@
+/* bender-tags: copyformatting */
+/* bender-ckeditor-plugins: wysiwygarea, toolbar, copyformatting, undo */
+/* bender-include: _helpers/tools.js*/
+/* bender-ui: collapsed */
+
+( function() {
+	'use strict';
+
+	var leftMouseButton = CKEDITOR.env.ie && CKEDITOR.env.version < 9 ? 1 : CKEDITOR.MOUSE_BUTTON_LEFT;
+
+	bender.editor = {
+		config: {
+			allowedContent: true
+		}
+	};
+
+
+	bender.test( {
+		// (#2780)
+		'test basic undo integration': function() {
+			var editor = this.editor,
+				bot = this.editorBot,
+				sel = editor.getSelection(),
+				el1,
+				el2;
+
+			resetEditorStatusAndCheckBasicAsserts( bot );
+
+			el1 = editor.editable().findOne( '#one' );
+			el2 = editor.editable().findOne( '#two' );
+
+			for ( var i = 0; i < editor.undoManager.limit; i++ ) {
+				sel.selectElement( i % 2 ? el1 : el2 );
```

Comment:
It can be simplified by getting all `span` elements using `editor.editable().find` and then

```suggestion
				sel.selectElement( spans.getItem( i % 2 ) );
```

No need for creating two separate variables.